front/components/sparkle/Notification.tsx

```diff
@@ -99,41 +91,12 @@ export function Notification({ title, description, type }: NotificationType) {
       leaveFrom="translate-y-0 opacity-100"
       leaveTo="translate-y-16 opacity-0"
     >
-      <div className="pointer-events-auto flex items-start gap-3 rounded-md border border-structure-100 bg-structure-0 p-2 shadow-md">
```

Comment:
Looks like this if ok for you
![image](https://github.com/dust-tt/dust/assets/5437393/6965adc9-725e-4890-9ad6-0bc7022a4706)

![image](https://github.com/dust-tt/dust/assets/5437393/8b806eaf-8799-43bb-ba1c-52d40f002b47)

Reply:
Yes, look's I know about :)
What i could not test is if they are properly triggered and dissmissable

Reply:
yep triggers & dismisses correctly